packages/ERTP/src/extentOps/helpers.js

```diff
@@ -0,0 +1,65 @@
+import harden from '@agoric/harden';
+import { passStyleOf } from '@agoric/marshal';
+import { insist } from '@agoric/insist';
+
+const makeObjListExtentOps = (customInsistFn, getIdentifierFn) => {
+  const identity = [];
+  const extentOps = harden({
+    insistKind: list => {
+      insist(passStyleOf(list) === 'copyArray')`list must be an array`;
+      list.forEach(customInsistFn);
+    },
+    empty: _ => identity,
```

Comment:
It looks like this hands out a shared mutable array. What ensures this can't be modified by clients?

Reply:
Good point. Let's harden it